tests/test_flowlogs_reader.py

```diff
@@ -35,6 +35,8 @@
 )
 
 
+SKIP_RECORD = -1
+
```

Comment:
This is a redefined constant - you can just import it from `flowlogs_reader.py`.